pkg/storage/couchdb/couchdbstore.go

```diff
@@ -346,6 +363,38 @@ func (i *couchDBResultsIterator) Value() ([]byte, error) {
 	return value, nil
 }
 
+func (c *CouchDBStore) getAllKeyValuePairs(rows *kivik.Rows) (map[string][]byte, error) {
+	allKeyValuePairs := make(map[string][]byte)
+
+	nextOK := rows.Next()
+	for nextOK {
```

Comment:
nit: `nextOK` is not necesary... the normal idiom is `for rows.Next() {`

Reply:
Done.